packages/backend/src/studio.ts

```diff
@@ -49,6 +49,7 @@ import { RecipeManager } from './managers/recipes/RecipeManager';
 import { GPUManager } from './managers/GPUManager';
```

Comment:
I think we should create a dedicated instrulab-api-impl.ts object, and also registering it in the same way we are doing for studio-api-impl. 

As we do not know yet the implication and complexity of instrutlab implementation, it is probably better to keep it appart.

Reply:
You mean to also split the `SutdioAPI`, by having a separate `InstructlabAPI`?

Reply:
Yeah !

Reply:
ok, I'll make the change